packages/govuk-frontend/src/govuk/govuk-frontend-component.mjs

```diff
@@ -49,7 +49,8 @@ export class GOVUKFrontendComponent {
   /**
    * Validates whether components are supported
    *
-   * @private
+   * @protected
+   * @static
    * @throws {SupportError} when the components are not supported
    */
   checkSupport() {
```

Comment:
**issue** This only documents the method as `static`, but the method itself is not `static` from a JavaScript point of view (it's interesting that TypeScript does not pick up on that, actually, that'll be something to remember).

```suggestion
   * @protected
   * @throws {SupportError} when the components are not supported
   */
  static checkSupport() {
```

Having the method static will help avoid overloading the console with errors when a component is repeated multiple times on the page (like the Copy button on the Design System page), letting `createAll` do a single initial check before looping.

Reply:
Actually, if we want to be able to call it from `createAll`, it needs to be not `@protected`, completely didn't think of that 😅 

```suggestion
   * @throws {SupportError} when the components are not supported
   */
  static checkSupport() {
```